.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ts-graphviz/reviewer
```

Comment:
@kamiazya 
It looks like an error has occurred, but is this a problem?
![image](https://github.com/user-attachments/assets/4c6ac95e-0250-4aa3-b287-b9282c1df493)

Reply:
@nagasawaryoya 
Thanks, I fixed the repository settings and the error seems to have disappeared.